rules/windows/process_creation/proc_creation_win_susp_copy_system_dir.yml

```
@@ -41,10 +42,7 @@ detection:
               - 'robocopy.exe'
               - 'XCOPY.EXE'
     target:
-        CommandLine|contains:
-            - '\System32'
-            - '\SysWOW64'
-            - '\WinSxS'
+        CommandLine|re: (?i)\s[\s'"]?C:\\Windows\\((System32|SysWOW64|WinSxS))
```

```suggestion
        CommandLine|re|i: \s['"]?C:\\Windows\\(System32|SysWOW64|WinSxS)
```
- see https://github.com/SigmaHQ/sigma-specification/blob/main/appendix/sigma-modifiers-appendix.md#regular-expression
- I don't see why the beginning needs to match `\s\s?`
- remove redundant grouping